### ISSUE TITLE
react-redux 7 bug return exact props type component

### ISSUE THREAD

**Issue by maystrenkoYurii** [flow-typed/flow-typed#3788]
**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Local Environment Information**
If this issue is specific to the CLI, provide the results of running `npx envinfo` from the command line in a GitHub gist.

**Comment by maystrenkoYurii** [comment]
But OwnProps should be non exact type
Finish fix which also corrects the previous error
![Снимок экрана от 2020-03-22 18-20-06](https://user-images.githubusercontent.com/23016311/77254469-ce7ebe00-6c69-11ea-9136-1be7b7b8abb2.png)

**Comment by maystrenkoYurii** [comment]
Access danied
![Снимок экрана от 2020-03-22 18-52-30](https://user-images.githubusercontent.com/23016311/77255189-59fa4e00-6c6e-11ea-996a-b96530b538b8.png)

**Comment by jedwards1211** [comment]
@maystrenkoYurii you need to fork this repository and push to your own fork, then you can make a pull request

**Comment by jedwards1211** [comment]
To do this using the `hub` CLI:
* install https://github.com/github/hub
* Then:
```sh
cd ~/flow-typed
hub fork
hub push --set-upstream maystrenkoYurii react-redux-fix-issue-3788
hub pull-request
```

**Comment by maystrenkoYurii** [comment]
@jedwards1211 
sorry but ci build fail and I don't understand why

**Comment by jedwards1211** [comment]
Yeah, the `react-redux` types are so complicated I don't know if I can figure it out either.  You can run the same tests that CI is running in your local copy by doing this:
```sh
cd ~/flow-typed/cli
yarn
node dist/cli.js run-tests react-redux
```
How come you're using `$Exact` in
```
return connect<Props, $Exact<OwnProps>, ...
```
I assume you get some other errors if you don't use `$Exact` there?

**Comment by maystrenkoYurii** [comment]
@jedwards1211  i am run test local without changes node v12 but its fail
add only \n end fail
![Снимок экрана от 2020-03-25 21-22-07](https://user-images.githubusercontent.com/23016311/77576994-bf547600-6ede-11ea-8ba2-11542193370e.png)

**Comment by AndrewSouthpaw** [comment]
Looks like the `react-redux_v7` tests are indeed broken in master.

**Comment by maystrenkoYurii** [comment]
@jedwards1211 
before change
![Снимок экрана от 2020-03-26 21-06-39](https://user-images.githubusercontent.com/23016311/77686635-ef1a8100-6fa5-11ea-94ed-babdb00d9f80.png)
![Снимок экрана от 2020-03-26 21-09-01](https://user-images.githubusercontent.com/23016311/77686723-0e191300-6fa6-11ea-9885-075a9e156579.png)
![Снимок экрана от 2020-03-26 21-10-14](https://user-images.githubusercontent.com/23016311/77686918-51738180-6fa6-11ea-97ff-dae3d72232ac.png)
after chenge
